src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -63,64 +66,79 @@ using ::testing::Return;
 using ::testing::ReturnRef;
 
 namespace {
-const int32_t kCommandId = 1;
-const uint32_t kAppId = 1u;
-const uint32_t kCmdId = 1u;
 const uint32_t kConnectionKey = 2u;
+const uint32_t kAppId = 1u;
+const uint32_t kMenuPosition = 10u;
+const uint32_t kCommandId = 1u;
+const uint32_t kCommandIcon = 1u;
+const uint32_t kVrPosition = 1u;
+const std::string kMenuName = "LG";
+const std::string kCommandIconValue = "10";
+const std::string kVrTitle = "vr_help_title";
+const std::string kVrText = "vr_help";
+const std::string kVrCommand = "vr_command";
+const std::string kHelpPrompt = "help_promt";
+const std::string kValidString = "correct_sting";
+const std::string kNotValidString = "wrong\t\nstring\\n";
 }  // namespace
 
 class SetGlobalPropertiesRequestTest
     : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
   SetGlobalPropertiesRequestTest()
       : mock_message_helper_(*MockMessageHelper::message_helper_mock())
-      , mock_app_(CreateMockApp()) {}
-
-  MessageSharedPtr CreateFullParamsUISO() {
-    MessageSharedPtr msg = CreateMessage(smart_objects::SmartType_Map);
-    (*msg)[am::strings::params][am::strings::connection_key] = kConnectionKey;
-    smart_objects::SmartObject menu_params =
-        smart_objects::SmartObject(smart_objects::SmartType_Map);
-    menu_params[am::strings::position] = 10;
-    menu_params[am::strings::menu_name] = "LG";
-
-    smart_objects::SmartObject msg_params =
-        smart_objects::SmartObject(smart_objects::SmartType_Map);
-    msg_params[am::strings::cmd_id] = kCmdId;
-    msg_params[am::strings::menu_params] = menu_params;
-    msg_params[am::strings::app_id] = kAppId;
-    msg_params[am::strings::cmd_icon] = 1;
-    msg_params[am::strings::cmd_icon][am::strings::value] = "10";
-    msg_params[am::strings::vr_help_title] = "vr_help_title";
-    msg_params[am::strings::vr_help][0][am::strings::text] = "vr_help";
-    msg_params[am::strings::vr_help][0][am::strings::position] = 1u;
-    msg_params[am::strings::help_prompt][0][am::strings::text] = "help_promt";
-    (*msg)[am::strings::msg_params] = msg_params;
-
-    return msg;
-  }
-
-  void SetUp() OVERRIDE {
-    ON_CALL(app_mngr_, application(kConnectionKey))
-        .WillByDefault(Return(mock_app_));
+      , mock_app_(CreateMockApp()) {
+    Mock::VerifyAndClearExpectations(&mock_message_helper_);
+    ON_CALL(app_mngr_, application(_)).WillByDefault(Return(mock_app_));
     ON_CALL(*mock_app_, app_id()).WillByDefault(Return(kConnectionKey));
     ON_CALL(app_mngr_, hmi_interfaces())
         .WillByDefault(ReturnRef(hmi_interfaces_));
   }
 
-  void TearDown() OVERRIDE {
+  ~SetGlobalPropertiesRequestTest() {
```

Comment:
@okozlovlux Judging by the content of SetUp and TearDown functions I decided that the same logic can be done by constructor and destructor, so I moved content of those functions to ctor and dtor.